hyper/http11/connection.py

```diff
@@ -83,17 +106,21 @@ def __init__(self, host, port=None, secure=None, ssl_context=None,
         self.ssl_context = ssl_context
         self._sock = None
 
+        # Keep request methods in a queue in order to be able to know
+        # in get_response() what was the request verb.
+        self._request_methods = []
```

Comment:
Yeah, I was thinking about pipelining while considering a list here.
OK, I will turn this to a single value